### PR TITLE
feat: production ready template

### PR DESCRIPTION
**Pull request by mrcpvn**

### CODE REVIEW

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: a3699256c0fb7d0e2343f4a951716374623104c0
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f8b5c1df-20ad-4efd-a7de-9c251470350a%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: src/WWTelematics.TenantService/serverless.template [hunk 1]
```diff
@@ -1,40 +1,213 @@
...
+        Runtime: dotnetcore2.1
+        Timeout: 30
+        CodeUri:
+          Fn::If:
+            - LocalDev
+            - !Ref AWS::NoValue
+            - |
+                Bucket: !Sub ${CodeUriBucket}
+                Key: !Sub ${CodeUriKey}
+                KmsKeyArn: !Ref KmsKey
```
Comment (davidw-foxtrax) [inline src/WWTelematics.TenantService/serverless.template]: KmsKeyArn should be a sibling of CodeUri and VpcConfig.

Comment (mrcpvn) [reply src/WWTelematics.TenantService/serverless.template]: fixed

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 83bc61f3bc1e1cff14ca0e1aeb7456f04014f02b
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=737f39ff-8c4e-422a-ab11-b9f51f9914d1%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by WW-CICD** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: pr-CodeBuild-WWTelematicsTenantService-CodeBuildProject
* Commit ID: 5c40eabd2da524d5921b11086a17989a1ba12691
* Result: SUCCEEDED
* [Build Logs](https://tlboygrlvi.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3515d0f8-a5c4-4c0c-ae34-811014133c47%2Fbuild.log) (available for 10 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**